### DISCUSSION TITLE
Track 1 does not match: level 40= 50

### DISCUSSION THREAD

**Discussion by weiyingjun122** [mifi/lossless-cut#2106] [General]
Track 1 does not match: level 40= 50

**Comment by mifi** [comment]
Hi. does it cause any problems?

**Reply by weiyingjun122** [reply]
Error reported when merging video files：Track 1 mismatch: time_base 1/90000= 1/12800

**Reply by mifi** [reply]
it means that the files are slightly different and merging might cause problems (but not necessarily - you have to try and see)

**Reply by fy860825** [reply]
> it means that the files are slightly different and merging might cause problems (but not necessarily - you have to try and see)

Do we have a solution for this ?

**Reply by mifi** [reply]
> Do we have a solution for this ?

you just have to try and see if it works. There isn't any way of removing the warning message.

**Reply by fy860825** [reply]
thanks for responding. got chatgpt to do some ffmpeg script to fix it with the provided mismatch message.
